Documentation/contributing/docs/docsframework.rst

```diff
@@ -0,0 +1,392 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _docs_framework:
+
+***********************
+Documentation framework
+***********************
+
+This page contains notes on the framework in use for Cilium documentation. Its
+objective is to help contributors understand the tools and build process for
+the documentation, and to help maintain it.
+
+Alas, this sort of document goes quickly out of date. When in doubt of
+accuracy, double-check the codebase to verify information. If you find
+discrepancies, please update this page.
+
+Sphinx
+======
+
+Cilium relies on `Sphinx`_ to generate its documentation.
+
+.. _Sphinx: https://www.sphinx-doc.org
+
+Sphinx features
+---------------
+
+Here are some specific Sphinx features in Cilium's documentation:
+
+- `Tab groups`_
+
+- `OpenAPI`_ documentation generation
+
+- Mark-up languages: reStructuredText (rST) and Markdown (`MyST`_ flavor)
+
+- Substitutions, for example:
+
+  - ``|SCM_WEB|``
+  - ``|CHART_VERSION|``
+
+- Multiple versions (for all supported branches, plus two aliases: ``stable``
+  and ``latest``)
+
+.. _OpenAPI: https://github.com/sphinx-contrib/openapi
+.. _Tab groups: https://github.com/executablebooks/sphinx-tabs/
+.. _MyST: https://myst-parser.readthedocs.io
+
+Sphinx version
+--------------
+
+The version of Sphinx in use is defined in
+``Documentation/requirements-min/requirements.txt``. For more details, see the
+:ref:`section on requirements <docs_requirements>`.
+
+Auto-generated contents
+=======================
+
+Some contents are automatically generated at build time. The documentation
+``Makefile`` contains the following target, shown here in a simplified version,
```

Comment:
should we just specify the path of the makefile i.e. `./documentation/Makefile` to avoid confusion?

Reply:
OK, I added it, thank you